CMakeLists.txt

```
@@ -6,6 +6,11 @@ PROJECT (msgpack-cxx LANGUAGES CXX)
 
 ADD_LIBRARY (msgpack-cxx INTERFACE)
 
+IF (CMAKE_VERSION VERSION_GREATER_EQUAL 3.19)
+    FILE(GLOB_RECURSE SOURCE_FILES include/*hpp)
```

```suggestion
    FILE(GLOB_RECURSE SOURCE_FILES include/*.hpp)
```

I read https://cmake.org/cmake/help/v3.7/command/file.html globbing-expressions examples.
It seems that *.hpp can avoid unexpected match.